cmd/export.go

```diff
@@ -60,6 +53,7 @@ new dataset, use --blank.`,
 	cmd.Flags().StringVarP(&o.BodyFormat, "body-format", "", "", "format for dataset body. default is the original data format. options: json, csv, cbor")
 	cmd.Flags().BoolVarP(&o.NoBody, "no-body", "b", false, "don't include dataset body in export")
 	cmd.Flags().BoolVarP(&o.PeerDir, "peer-dir", "d", false, "export to a peer name namespaced directory")
+	cmd.Flags().BoolVarP(&o.Zipped, "zipped", "z", false, "export as a zip file")
```

Comment:
`zipped` or `zip`? Is there prior art for `zipped`?

Reply:
I'm fine with `zip`, went with `zipped` only because that's what the old Options struct was using, but again I'm okay with changing it.